### PR TITLE
Self-selecting to person-of-color.

### PR DESCRIPTION
**Pull request by nstielau**
I liked the idea of letting people self-select into 'person of color', which would let us roll-up different racial/ethnic minorities.  Also added what seems to be a fairly standard list of ethnicities, pulled from http://www.informationisbeautiful.net/visualizations/diversity-in-tech/

ping @drnikki

### CODE REVIEW

File: SURVEY.md [hunk 1]
```diff
@@ -48,11 +48,20 @@ How do we want people to indicate their language?
...
+    Do you identify as a person of color? (e.g. African, Latino/Hispanic, Asian, South Asian, Native American, Middle Eastern, Pacific Islander [1])
 
-    Choose the racial identities (phrasing!) with which you identify
+    () Yes
+    () No
+
+    With which ethnicity do you most identify (from Information is Beautiful [2])?
 
-    - combo box of words (tbd)
+    () White
```
Comment (drnikko) [inline SURVEY.md]: often people identify as more than one racial group.  We should consider making these checkboxes.

Comment (nstielau) [inline SURVEY.md]: Definitely. I think 'Multi' and 'Other' will allow for that, while helping to keep our data clean and easy.

Comment (drnikko) [inline SURVEY.md]: I'd like to hear from some people of color before this gets merged - or we can at least change the () to [] to let people choose more than one without the 'multi' box.

**Comment by nstielau** [comment]
Ok @drnikki, made it checkboxes.  Can you merge?

**Pull request merged**